operations/operations/publishing/publishing-guidelines/documentation-style-guide.md

```diff
@@ -63,12 +63,32 @@ We use the Oxford \(serial\) comma in our documentation.
 
 ### Capitalization
 
+Capitalization is a work in progress and inconsistencies are actively being worked on. New documentation and UI/in-product content should follow this style guide.
+
+#### Documentation headings
+
+**H1**
```

Comment:
I'm just curious if the "H1/H2" terminology is clear. Instead, we could use "Page Heading", "All Other Headings"

Reply:
Good point, I agree.